cmd/repl/repl.go

```diff
@@ -119,6 +137,31 @@ func main() {
 		if isTable && (table.NumLines() > 0 || !tableRendered) {
 			table.Render()
 		}
-		fmt.Println("")
+	}
+
+	if flagPassed("execute", "e") {
+		sqlRun(*cliStmt)
+	} else {
+		sqlFile := os.Stdin
+		if flagPassed("file", "f") {
+			sqlFile, err = os.Open(*sqlFileName)
+			if err != nil {
+				log.Fatal(err)
+			}
+			defer sqlFile.Close()
+		}
+		scanner := bufio.NewScanner(sqlFile)
+
+		for {
```

Comment:
I propose to extract this `for` loop out from `main` to be a function definition

```go
def repl(inputScanner bufio.Scanner, isTerminal bool)
```

Reply:
Done.